app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -1,115 +0,0 @@
-package de.danoeh.antennapod.fragment;
-
-import android.content.Context;
-import android.content.SharedPreferences;
-import android.os.Bundle;
-import android.view.Menu;
-import android.view.MenuItem;
-import android.view.View;
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-import com.joanzapata.iconify.Iconify;
-import de.danoeh.antennapod.R;
-import de.danoeh.antennapod.model.feed.FeedItem;
-import de.danoeh.antennapod.model.feed.FeedItemFilter;
-import de.danoeh.antennapod.core.storage.DBReader;
-import de.danoeh.antennapod.dialog.FilterDialog;
-import org.apache.commons.lang3.StringUtils;
-
-import java.util.List;
-import java.util.Set;
-
-/**
- * Like 'EpisodesFragment' except that it only shows new episodes and
- * supports swiping to mark as read.
- */
-public class AllEpisodesFragment extends EpisodesListFragment {
```

Comment:
I'm not really happy about this huge amount of unrelated changes, to be honest...

Reply:
Hmm :(
I think I only changed it from TabPagerFragment or whatever it was before, to the single list layout from AllFragments and included the toolbar in there - I guess the toolbar could be in a separate fragment but that seems unnecessarily nested, no?